### DISCUSSION TITLE
RAM memory for Scenic+

### DISCUSSION THREAD

**Discussion by MohammadIzadi** [aertslab/scenicplus#221] [Q&A]
Hi,

Thank you for developing SCENIC+. I am excited to use it.
I have one question. I want to know is 52Gb of RAM memory sufficient for run Scenic+, pyscisTarget and pycisTopic?. My data approximately has 3000 cells for each scRNA-seq and scATAC-seq and 1890 genes in scRNA-seq data and 566400 regions for scATAC-seq data.

Sincerely,

**Comment by SeppeDeWinter** [comment]
Hi @MohammadIzadi 

I'm afraid that you might run out of memory (certainly when analysing the scATAC-seq data). We usually use 5 times as much memory.

Best,

Seppe
